hyperjumptech/monika
# fix: iphone se 2 visual issues on zoomed ui

This PR resolves #265 

### How did you do it
1. Fix breaking words on top banner and demo banner
2. Fix box decoration break on gradient text
3. Adjust npm install command auto-select to `select-all`
4. Adjust main banner button positioning to flex column on medium screen (768 px)
5. Add copyright text on footer

### How to test
1. `cd docs`
2. `npm run dev`
3. Visit `http://your-local-ip:3000` on iOS SE 2 with zoomed UI
4. Also need to check other impact on different screen sizes
